src/Formatter/IntlMoneyFormatter.php

```diff
@@ -16,13 +16,19 @@
      * @var \NumberFormatter
      */
     private $formatter;
+    /**
+     * @var int|null
```

Comment:
null is casted to 0 if used in arithmetical expressions. Shouldn't 0 be the default value?